### ISSUE TITLE
Checkpoints for ETH5 and InD

### ISSUE THREAD

**Issue by HRHLALALA** [luigifilippochiara/Goal-SAR#2]
Hi, I have downloaded the datasets from your script `download_checkpoints.sh` but only the SDD version is available. Can you share the checkpoints for ETH5 and InD as well? Thanks.

**Comment by luigifilippochiara** [comment]
Hi @HRHLALALA thank you for your interest in our project. For the time being we only provided support for the sdd dataset. We are working on releasing the others. Alternatively, you can train the model directly to reproduce the same results

**Comment by HRHLALALA** [comment]
Hi, thanks for your reply. May I ask how many epochs and batch size did you use for training the UNIV dataset? Also, I attached my hyperparameters here:

`python main.py --model_name Goal_SAR --phase train_test --num_epochs 500 --batch_size 32 --validate_every 1 --learning_rate 0.0001 --down_factor 4 --dataset eth5 --test_set=univ --use_wandb=True --down_factor=4`

```
add_noise_traj: true
base_dir: .
batch_size: 32
clip: 1
compute_test_nll: true
compute_valid_nll: false
config: ./output/univ/Goal_SAR/config_train_test.yaml
data_augmentation: true
dataset: eth5
device: cuda:0
down_factor: 4
fast_debug: false
fast_debug_num: 3
learning_rate: 0.0001
load_checkpoint: null
model_dir: ./output/univ/Goal_SAR
model_name: Goal_SAR
num_epochs: 500
num_test_runs: 5
num_test_samples: 20
num_valid_samples: 20
obs_length: 8
optimizer: Adam
phase: train_test
pred_length: 12
reproducibility: true
sampler_temperature: 1
save_base_dir: output
save_dir: ./output/univ
save_every: 60
scheduler: null
seq_length: 20
shift_last_obs: true
shuffle_test_batches: true
shuffle_train_batches: true
skip_ts_window: 5
start_validation: 5
test_set: univ
traj_normalization: 16
use_cuda: true
use_ttst: true
use_wandb: true
validate_every: 1
``` 
Now I trained the model at epoch 256 which takes me two days and the current results are 
`----Epoch 256, time/epoch=6 min and 45 sec, learning_rate=0.00010, train_traj_MSE_loss=0.47110, train_goal_BCE_loss=0.00354, valid_traj_MSE_loss=68.58920, valid_goal_BCE_loss=0.00460, train_ADE=2.604, train_FDE=0.000, valid_ADE=13.014, valid_FDE=23.342, valid_ADE_world=0.288, valid_FDE_world=0.516
best_ADE=12.399, best_FDE=20.975, best_ADE_world=0.276, best_FDE_world=0.465, best_goal_BCE_loss=0.004, best_epoch_ADE=123, best_epoch_FDE=96, best_epoch_ADE_world=123, best_epoch_FDE_world=96, best_epoch_goal_BCE_loss=203`

Can you help check whether these are consistent with your training for the final version? Thanks!

**Comment by luigifilippochiara** [comment]
Hi, I checked your parameters, and they seem exactly the ones we used in the final version. Also your metrics (best_ADE_world=0.276, best_FDE_world=0.465) are in line with our results. One thing to keep in mind is that we experienced some variability between the different runs, so if you run the code again you will see (slightly) different results.

We indeed use --num_epochs 500 --batch_size 32. Alternatively, you can try to experiment with the following set of hyperparameters: --down_factor=[4,8], --batch_size [16,32,64], --learning_rate [0.005, 0.0001, 0.00015]. Hope this helps

**Comment by HRHLALALA** [comment]
Thanks for your confirmation. May I ask approximately at which epoch will FDE=0.43 occurs? It needs quite a long time to train so I hope I can rerun it in advance if it cannot reach the paper result. I have already conducted a full train but the metrics are stuck at `best_ADE_world=0.276, best_FDE_world=0.46`.

**Comment by luigifilippochiara** [comment]
Unfortunately the process is quite stochastic so multiple runs may be required. The minimum is sometimes due to a random oscillation. In our experiments, we usually obtained the best results around epochs 300-500. Changing the random seed may also help
